### PR TITLE
Random CMR fixes

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
Collection of small CMR fixes including:

Primary Nav
- Move NodeBalancers from Compute to Network
- Rename `Virtual VLANS` to `Virtual VLANs`
- Add hover state and remove animation for settings icon
- Remove compact toggle option from settings

Linode Landing
- Marking linodes being cloned as busy

Etc.
- Fix content shifting to the left when modal is open (only occurs on linode grid view, see comment)
- Remove button transition / animation

## Type of Change
- Non breaking change ('update', 'change')

### CODE REVIEW

File: packages/manager/src/index.css [hunk 1]
```diff
@@ -14,7 +14,7 @@ html {
 body {
   height: 100%;
   margin: 0;
-  padding: 0;
+  padding: 0 !important;
```
Comment (tiffwong) [inline packages/manager/src/index.css:17]: Seems to be part of a longstanding issue that is hard to reproduce, adding `!important` should reinforce the padding and prevent the content from shifting over when a modal is opened on linode grid view

File: packages/manager/src/mocks/testBrowser.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { handlers, mockDataHandlers } from './serverHandlers';
  */
 if (MOCK_SERVICE_WORKER) {
   const worker = setupWorker(...handlers);
-  // worker.start();
+  worker.start();
```
Comment (tiffwong) [inline packages/manager/src/mocks/testBrowser.ts:14]: Accidentally merged a PR with this commented out

File: packages/manager/src/index.css (hunk 1 above)
Comment (tiffwong) [reply packages/manager/src/index.css:17]: See ticket in JIRA, M3-3638

**Review by WilkinsKa1 — APPROVED** [review]

**Review by DevDW — APPROVED** [review]
LGTM 🚢

File: packages/manager/src/index.css (hunk 1 above)
Comment (johnwcallahan) [reply packages/manager/src/index.css:17]: FYI this is reproducible by going to your Mac's Settings -> General -> Show scroll bars: "Automatically based on mouse or trackpad".

File: packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx [hunk 3]
```diff
@@ -694,18 +695,22 @@ export class ListLinodes extends React.Component<CombinedProps, State> {
...
   return linodes.filter(thisLinode => {
-    const displayStatus = mapLinodeStatus(thisLinode.status);
+    const displayStatus = mapLinodeStatus(thisLinode.status, thisLinode.event);
     return displayStatus === status;
   });
 };
 
-const mapLinodeStatus = (linodeStatus: LinodeStatus) => {
+const mapLinodeStatus = (linodeStatus: LinodeStatus, linodeEvent?: Event) => {
+  if (linodeEvent) {
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:710]: I don't think this quite does what we want:

Screen Shot 2020-09-28 at 9 22 39 PM

I really do have 3 running Linodes here, so the most accurate thing is the the green indicator to the left of the Linode Label. 

I know I mentioned the status of Linodes being cloned was strange, but after we talked about it, I'm thinking it probably makes more sense as-is. This Linodes are not "Pending" in the sense that they can can still be shut down, etc.

**Review by Jskobos — APPROVED** [review]

File: packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx (hunk 3 above)
Comment (tiffwong) [reply packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:710]: Hmm should I also revert the changes that apply the busy icon to the linodes are cloning?

Comment (johnwcallahan) [reply packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:710]: The screenshot above is from this PR in the Linodes List view, and still have the "Running" icon... which view do you mean?

Comment (tiffwong) [reply packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:710]: In [https://github.com/linode/manager/pull/6912](https://github.com/linode/manager/pull/6912), the statuses for cloning are changed to busy

Screen Shot 2020-09-30 at 9 58 32 AM

Comment (johnwcallahan) [reply packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:710]: Yeah, that's what I meant about Linodes that are being cloned not actually "Busy" in the sense that they can still be shut down, etc. I hadn't considered that when I left that comment. I think the conclusion we came to in demo prep was to leave them as whatever the actual `linode.status` is (which means we wouldn't need to look at the recent event).

 I'm all for other suggestions though... the green icon + the "Cloning to:" message does look a little strange. In Prod currently we show the spinning "Busy" indicator, but things are a bit different here in CMR. (This is totally something we can come back to though, if you want to revert those changes and move on.)

Comment (tiffwong) [reply packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.tsx:710]: Hmm I'll bring it up during standup to see if there's an alternative way to handle this 🤔

**Review by johnwcallahan — APPROVED** [review]

**Pull request merged**